src/styles/index.scss

```diff
@@ -1,3 +1,156 @@
-body {
+// Змінні
+$font-family: roboto, helvetica, sans-serif;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$link-color: #00acdc;
+
+// Стилі для базових HTML елементів
+body,
+div,
+p,
+a {
   margin: 0;
+  box-sizing: border-box;
+  font-family: $font-family;
+}
+
+// Стилі для контейнера
+.container {
+  display: flex;
+  width: 200px;
+  height: 406px;
+
+  // Стилі для картки
+  .card {
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+
+    // Стилі для зображення в картці
+    .card__image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+
+    // Стилі для інформації в картці
+    .card__information {
+      margin-top: 40px;
+
+      // Стилі для заголовка
+      .card__title {
+        color: $primary-color;
+        font-size: 12px;
+        font-weight: 500;
+        line-height: 18px;
+        margin-bottom: 4px;
+      }
+
+      // Стилі для коду товару
+      .card__code {
+        color: $secondary-color;
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
```

Comment:
instead of describing font-size, -weigth, line-heigth for each selector, try to make default variable (as you did with colors) and use it, changing this parameters only if need